cmd/xl-v1-healing.go

```diff
@@ -42,6 +42,9 @@ func (xl xlObjects) HealFormat(ctx context.Context, dryRun bool) (madmin.HealRes
 // `policy.json, notification.xml, listeners.json`.
 func (xl xlObjects) HealBucket(ctx context.Context, bucket string, dryRun, remove bool) (
 	result madmin.HealResultItem, err error) {
+	if !dryRun {
+		defer ObjectPathUpdated(bucket)
```

Comment:
I am not sure why updating for healing is needed, since heal won't change the object size

Reply:
There may have been disks crawled that didn't contain all of the objects. Remember, we only crawl one disk per set.

Reply:
> There may have been disks crawled that didn't contain all of the objects. Remember, we only crawl one disk per set.

oh yes, you are right